classes/class-wc-connect-shipping-label.php

```diff
@@ -157,6 +157,7 @@ protected function get_form_layout() {
 				'type' => 'summary',
 				'tab_title' => 'Preview',
 				'title' => 'dfhdfghdfgh',
```

Comment:
How will this be used? Go ahead and put something like `__( 'TODO Title', 'woocommerce' )` now